src/ooxml/java/org/apache/poi/xssf/usermodel/XSSFCell.java

```diff
@@ -1311,4 +1312,23 @@ void notifyArrayFormulaChanging(){
                 "You cannot change part of an array.";
         notifyArrayFormulaChanging(msg);
     }
+    
+    /***
+     * Moved from XSSFRow.shift(). Not sure what is purpose. 
```

Comment:
this doesn't read like public javadoc - could you use // comments for this sort of comment?

Reply:
Sure, did it.
But I'd like someone to review method (it was a snippet inside other method) and write a real comment for it.
Anyway, I'll remove my comment once we're done with this patch.